indico/modules/events/registration/templates/display/participant_list.html

```diff
@@ -4,6 +4,10 @@
     {% trans %}Participant List{% endtrans %}
 {% endblock %}
 
+{% block subtitle %}
+    {% trans num=registrations|length %}{{ num }} participants{% endtrans %}
```

Comment:
I know how to do it, I was just wondering for the wording. I still don't line "One participant" as a subtitle...